src/index.js

```diff
@@ -3,7 +3,18 @@ import ReactDOM from 'react-dom';
 import 'typeface-roboto';
 import './index.css';
 import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import thunk from 'redux-thunk';
+import {Provider} from 'react-redux';
+import rootReducer from './redux/rootReducer';
+import {applyMiddleware, compose, createStore} from 'redux';
 
-ReactDOM.render(<App />, document.getElementById('root'));
-registerServiceWorker();
+const initialState = {};
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const store = createStore(rootReducer, initialState, composeEnhancers(applyMiddleware(thunk)));
+
```

Comment:
Die Zeilen mit _initialState_ und _store_ würde ich auslagern. Entweder in den rootReducer und dann _store.jsx_ entfernen oder in _store.jsx_ und dann den rootReducer nur dort importieren und den Store dann in das _index.js_ importieren.